stories/core/InformationPanel.stories.tsx

```diff
@@ -0,0 +1,256 @@
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import { CellProps } from 'react-table';
+import { action } from '@storybook/addon-actions';
+import { SvgEdit } from '@itwin/itwinui-icons-react';
+import {
+  InformationPanel,
+  InformationPanelProps,
+  InformationPanelWrapper,
+  Table,
+  Text,
+  LabeledTextarea,
+  Button,
+  IconButton,
+  LabeledInput,
+} from '../../src/core';
+import { CreeveyStoryParams } from 'creevey';
+
+export default {
+  component: InformationPanel,
+  subcomponents: { InformationPanelWrapper },
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    id: { control: { disable: true } },
+    headerActions: { control: { disable: true } },
+    children: { control: { disable: true } },
+  },
+  args: {
+    orientation: 'vertical',
+    resizable: true,
+  },
+  title: 'Core/InformationPanel',
+  parameters: {
+    docs: { source: { excludeDecorators: true } },
+    creevey: {
+      captureElement: '.iui-information-panel-wrapper',
+      tests: {
+        async open() {
+          const button = await this.browser.findElement({ css: '.iui-button' });
+          await button.click();
+
+          const opened = await this.takeScreenshot();
+          await this.expect({ opened }).to.matchImages();
+        },
+      },
+    } as CreeveyStoryParams,
+  },
+} as Meta<InformationPanelProps>;
+
+export const Basic: Story<InformationPanelProps> = (args) => {
+  const [openRowIndex, setOpenRowIndex] = React.useState<number>();
+
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: 'Table',
+        columns: [
+          { id: 'name', Header: 'Name', accessor: 'name' },
+          {
+            Header: 'Details',
+            Cell: ({ row: { index } }: CellProps<{ name: string }>) => (
+              <Button onClick={() => setOpenRowIndex(index)}>Details</Button>
+            ),
+          },
+        ],
+      },
+    ],
+    [],
+  );
+
+  const data = React.useMemo(
+    () =>
+      [...Array(10).fill(null)].map((_, index) => ({ name: `Row${index}` })),
+    [],
+  );
+
+  const lorem100 = `Lorem ipsum dolor sit, amet consectetur adipisicing elit. Accusamus veniam dicta error doloremque libero sit est. Voluptatum nam modi, ex illum veritatis nobis omnis porro quod harum optio minus magnam tenetur quia dolor quis natus, eius, suscipit hic? Nobis deleniti obcaecati, sequi mollitia vero magnam error quidem, voluptatem asperiores repudiandae, molestias sit et voluptatibus magni. Sequi delectus, sunt eaque corrupti architecto modi suscipit? Quos in itaque dolore voluptas saepe natus repellat ad qui dolores. Incidunt temporibus ut, unde maxime nam explicabo saepe aspernatur molestiae iste libero neque, alias corporis laboriosam fugiat ad. Dicta neque quos fuga odit quae sequi dolore!`;
+
+  return (
+    <InformationPanelWrapper>
```

Comment:
Would it shift the Table if there would be no wrapper? If yes, can we have a story with that?